.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
-  "extends": ["prettier"],
-  "plugins": ["prettier"],
+  "extends": ["prettier", "plugin:jsx-a11y/strict"],
```

Comment:
`"plugin:jsx-a11y/recommended"` would be better and less restrictive for a starting point

Reply:
How strict is strict? Because we're doing software that needs to be accessible, strict rules might be what we should be using.

Reply:
`recommended` stills notifies us with similar accessibility errors that `strict` does, except `recommended` gives us options to customize in a few cases (ref: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y#rule-strictness-in-different-modes)

Reply:
Idea here was to try strict rules first and we can fallback to recommended if they unnecessarily slow development down. It's easier to go that way than tighten the rules after changes has been applied.

Reply:
Checked a few other repos, and dvk seems to be using recommended. So based on that I'd say that it should be enough 🙂

Reply:
Changed setting to recommended. Thanks for mentioning this @NinaDang97!